cli/src/main.rs

```diff
@@ -242,7 +243,13 @@ fn do_main(matches: &ArgMatches<'_>) -> Result<(), Box<dyn error::Error>> {
         let (mut config, signers) = parse_args(&matches, wallet_manager)?;
         config.signers = signers.iter().map(|s| s.as_ref()).collect();
         let result = process_command(&config)?;
-        println!("{}", result);
+        let (_, submatches) = matches.subcommand();
+        let sign_only = submatches
+            .map(|m| m.is_present(SIGN_ONLY_ARG.name))
+            .unwrap_or(false);
+        if !sign_only {
```

Comment:
This part seems a little clunky to me, but I guess it will probably go away when I get around to https://github.com/solana-labs/solana/issues/7827

Reply:
I totally agree.  Unfortunately it's the only way to drop that JSON w/o modifying every call to a `process_*()` implementing offline signing. And the drop was pretty necessary after adding two more lists to said JSON :sweat_smile: 

#7827 is going to be a real treat judging by how arbitrarily we drop text all over the place :slightly_smiling_face:.  It does indeed look like it'll clear this area up